server/src/main/java/com/broadcom/lsp/cobol/service/CobolTextDocumentService.java

```diff
@@ -87,14 +92,16 @@
       Completions completions,
       Occurrences occurrences,
       DataBusBroker dataBus,
-      CodeActions actions) {
+      CodeActions actions,
+      CustomThreadPoolExecutor customThreadPoolExecutor) {
```

Comment:
Optional: this name seems very wordy, especially on the usages. Maybe, `executors` would be enough?

Reply:
done